CMakeLists.txt

```diff
@@ -196,6 +207,19 @@ target_include_directories(${PROJECT_NAME} PUBLIC
 	"${CMAKE_CURRENT_SOURCE_DIR}/src"
 )
 
+# Add autogen include dirs to include path, otherwise the includes are not found
+# when used in the header (via https://stackoverflow.com/q/44746308/)
```

Comment:
It seems this feature is CMake 3.9+, bumping minimum version from 3.7 is suggested.